atd-etl/app/process/helpers_import.py

```diff
@@ -352,8 +353,10 @@ def get_crash_record(crash_id):
     :return: string
     """
     # First generate a query with a list of the columns we care about
-    query = search_crash_query_full(crash_id=crash_id,
-                                    field_list=CRIS_TXDOT_COMPARE_FIELDS_LIST)
+    query = search_crash_query_full(
+        crash_id=crash_id,
+        field_list=CRIS_TXDOT_COMPARE_FIELDS_LIST+["apd_human_update"]
```

Comment:
I love that in python two merge two arrays all you need is the + sign. ✔️ 🙌

Reply:
Python magic 🎩🐇